docs/index.md

```diff
@@ -76,58 +76,107 @@ connection "alicloud" {
 }
 ```
 
-## Get involved
-
-* Open source: https://github.com/turbot/steampipe-plugin-alicloud
-* Community: [Slack Channel](https://steampipe.io/community/join)
-
-
-## Advanced configuration options
-
-For users with multiple accounts and more complex authentication use cases, here are some examples of advanced configuration options:
+## Multi-Account Connections
 
-### Specify multiple accounts 
-A common configuration is to have multiple connections to different accounts:
+You may create multiple alicloud connections:
 
 ```hcl
-connection "ali_account_aaa" {
+connection "alicloud_dev" {
   plugin      = "alicloud"
   secret_key  = "gMCYsoGqjfThisISNotARealKeyVVhh"
   access_key  = "ASIA42DZSWFYSN2PFHPJ"
-  regions     = ["us-east-1" , "ap-south-1"]
+  regions     = ["eu-central-1" , "cn-hangzhou"]
+}
+
+connection "alicloud_qa" {
+  plugin      = "alicloud"
+  secret_key  = "gMCYsoGqjfThisAintARealKeyVVhh"
+  access_key  = "ASIA42DZSWFYS42PFJHP"
+  regions     = ["cn-hangzhou"]
 }
 
-connection "ali_account_bbb" {
+connection "alicloud_prod" {
   plugin      = "alicloud"
   secret_key  = "gMCYsoGqjfThisAintARealKeyVVhh"
   access_key  = "ASIA42DZSWFYS42PFJHP"
-  regions     = ["cn-east-1"]
+  regions     = ["cn-hangzhou"]
 }
+```
+
+Each connection is implemented as a distinct [Postgres schema](https://www.postgresql.org/docs/current/ddl-schemas.html). As such, you can use qualified table names to query a specific connection:
+
+```sql
+select * from alicloud_qa.alicloud_account;
+```
+
+You can multi-account connections by using an [**aggregator** connection](https://steampipe.io/docs/using-steampipe/managing-connections#using-aggregators). Aggregators allow you to query data from multiple connections for a plugin as if they are a single connection:
 
+```hcl
+connection "alicloud_all" {
+  plugin      = "alicloud"
+  type        = "aggregator"
+  connections = ["alicloud_dev", "alicloud_qa", "alicloud_prod"]
+}
 ```
 
-### Specify static credentials using environment variables 
+Querying tables from this connection will return results from the `alicloud_dev`, `alicloud_qa`, and `alicloud_prod` connections:
+
+```sql
+select * from alicloud_all.alicloud_account;
+```
+
+Alternatively, can use an unqualified name and it will be resolved according to the [Search Path](https://steampipe.io/docs/guides/search-path). It's a good idea to name your aggregator first alphbetically, so that it is the first connection in the search path (i.e. `alicloud_all` comes before `alicloud_dev`):
```

Comment:
```suggestion
Alternatively, you can use an unqualified name and it will be resolved according to the [Search Path](https://steampipe.io/docs/guides/search-path). It's a good idea to name your aggregator first alphabetically, so that it is the first connection in the search path (i.e. `alicloud_all` comes before `alicloud_dev`):
```